.github/workflows/build-win.yml

```diff
@@ -44,3 +44,26 @@ jobs:
         TAG: superbench/main:win2004
         USER: ${{ secrets.DOCKERHUB_USERNAME }}
         PASS: ${{ secrets.DOCKERHUB_TOKEN }}
+  directx-unit-test:
+    name: DirectX unit test
+    needs: docker
+    runs-on: [self-hosted, windows, x64, win2004]
+    steps:
```

Comment:
remove? they should be in the same job to reuse latest built image

Reply:
why need to be in the same job, I think it's reuse the latest build image now since I have defined the dependency of these 2 jobs?